modules/navier_stokes/test/tests/finite_volume/ins/channel-flow/tests

```diff
@@ -23,6 +23,7 @@
     method = "!dbg"
     requirement = 'The system shall be able to model free-slip conditions in a channel using the NSFV action.'
     valgrind = HEAVY
+    recover = false
```

Comment:
Why?

Reply:
recovery is manually tested in `[checkpoint-restart]` below.